src/studio/src/designer/backend/Infrastructure/GitRepository/AltinnAppGitRepository.cs

```diff
@@ -203,6 +200,20 @@ public async Task SaveTextsV2(string languageCode, Dictionary<string, string> js
             await WriteTextByRelativePathAsync(textsFileRelativeFilePath, texts);
         }
 
+        /// <summary>
+        /// Overwrite or creates a markdown file for a specific text for a specific language.
+        /// </summary>
+        /// <param name="languageCode">Language identifier</param>
+        /// <param name="text">Keyvaluepair containing markdown text</param>
+        public async Task SaveTextMarkdown(string languageCode, KeyValuePair<string, string> text)
+        {
+            string fileName = $"{text.Key}.{languageCode}.texts.md";
+
+            var textsFileRelativeFilePath = Path.Combine(CONFIG_FOLDER_PATH, LANGUAGE_RESOURCE_FOLDER_NAME, MARKDOWN_TEXTS_FOLDER_NAME, fileName);
```

Comment:
Wasn't it decided to move the new text files out of the config folder and have texts/ on the root of the application?

Reply:
Keeping this change on standby until clarified